backend/entities/Models/ProjectFilter.cs

```diff
@@ -74,12 +79,14 @@ public ProjectFilter(Dictionary<string, Microsoft.Extensions.Primitives.StringVa
             var filter = new Dictionary<string, Microsoft.Extensions.Primitives.StringValues>(query, StringComparer.OrdinalIgnoreCase);
             this.ProjectNumber = filter.GetStringValue(nameof(this.ProjectNumber));
             this.Name = filter.GetStringValue(nameof(this.Name));
-            this.StatusId = filter.GetIntNullValue(nameof(this.StatusId));
+            this.StatusId = filter.GetIntArrayValue("status");
+            this.StatusId = filter.GetIntArrayValue(nameof(this.StatusId));
```

Comment:
are these two lines correct?

Reply:
They are, I decided it was likely someone would return `status` as the query parameter rather than the more verbose `statusId`.